<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QWidget" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_pro">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HGYT2_CNKI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tongji Hospital Calling System Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_clk">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>291</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Set the clock of the system</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>118</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lyt">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>440</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rage Italic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LYT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rage Italic</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>2023.08</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Text</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Get Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_doc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hos">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>500</width>
     <height>253</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_doc</zorder>
  <zorder>label_pro</zorder>
  <zorder>label_clk</zorder>
  <zorder>timeEdit</zorder>
  <zorder>label_lyt</zorder>
  <zorder>label_date</zorder>
  <zorder>pushButton_logo</zorder>
  <zorder>pushButton_start_2</zorder>
  <zorder>pushButton_hos</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
